src/components/EditorWidgets/Date/DateControl.js

```diff
@@ -36,9 +36,8 @@ export default class DateControl extends React.Component {
 
   handleChange = datetime => {
     const { onChange } = this.props;
-    if (!this.format || datetime === '') {
```

Comment:
We do still need to handle the empty case here, in case the user wants to empty the field.